### ISSUE TITLE
VRAM accumulation in Inference

### ISSUE THREAD

**Issue by sir-xabier** [WuJunde/MedSegDiff#105]
Hi,

I'm currently working with this fabulous framework for adapting it to another problem.

Even though the model is in evaluation mode I suggest including a with torch.no_grad(): when performing the inference process (internally in the DPM solver). I had problems with memory accumulation during the inference process and I have solved it this way.
with torch.no_grad(): 
                output = model(x, t_input, **model_kwargs)

**Comment by WuJunde** [comment]
check this issue https://github.com/WuJunde/MedSegDiff/issues/69
